gatsby-node.js

```diff
@@ -99,4 +106,253 @@ async function makeSynoptic(createPage, reporter, graphql) {
       context: section
     })
   }
+}
+
+async function makeSearchIndex(reporter, graphql){
+  const lunr = require('lunr')
+  const remark = (await import('remark')).remark
+
+  const jsdom = require("jsdom")
+  const { JSDOM } = jsdom
+
+  const result_md = await graphql(`
+    query {
+      allMarkdownRemark {
+        edges {
+          node {
+            frontmatter {
+              path
+              title
+            }
+            rawMarkdownBody
+          }
+        }
+      }
+    }
+  `)
+  const result_synoptic = await graphql(`
```

Comment:
synoptic means "shown at the same time"; the name was used for a query that collected two TEI documents to show them side by side on the same web page. In this case you're just loading TEI docs for indexing purposes so `result_tei` would be a better name (or something similar)